plugins/aea-cli-ipfs/aea_cli_ipfs/registry.py

```diff
@@ -161,7 +161,7 @@ def fetch_ipfs(
     if remote:
         ipfs_tool = IPFSTool(get_ipfs_node_multiaddr())
```

Comment:
Isn't this going to fail [here](https://github.com/valory-xyz/open-aea/blob/feat/default-ipfs-url/plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py#L111) if we do not have IPFS installed locally?